gefserver/pier/pier.go

```diff
@@ -146,6 +150,42 @@ func (p *Pier) BuildService(buildDir string) (db.Service, error) {
 	return service, nil
 }
 
+// startTimeOutTicker starts a clock that checks if a job exceeds an execution timeout
+func (p *Pier) startTimeOutTicker(jobId db.JobID) {
+	ticker := time.NewTicker(time.Second * time.Duration(p.checkInterval))
+	go func() {
```

Comment:
this goroutine is not necessary, there's another one when you're calling the startTimeOutTicker function